config/locales/en.yml

```diff
@@ -429,10 +429,13 @@ en:
     rent:
       min:
         title_text: "You told us the rent is %{brent}"
-        hint_text: "The minimum rent expected for this type of property in this local authority is £%{soft_min_for_period}."
+        hint_text: "The minimum rent expected for this type of property in this local authority is £%{soft_min_for_period}"
       max:
         title_text: "You told us the rent is %{brent}"
-        hint_text: "The maximum rent expected for this type of property in this local authority is £%{soft_max_for_period}."
+        hint_text: "The maximum rent expected for this type of property in this local authority is £%{soft_max_for_period}"
+    purchase_price:
+      title_text: "You told us the purchase price is %{value}"
```

Comment:
Can't we have `title_text: "You told us the purchase price is £%{value}"` instead? Would remove the need for prefixes

Reply:
But wouldn't this validation always be £? Cause it's price?

Reply:
have removed prefixes by using unit in number_to_currency 👍